presto-main/src/test/java/com/facebook/presto/sql/TestExpressionInterpreter.java

```diff
@@ -158,6 +160,9 @@ public void testComparison()
         assertOptimizedEquals("bound_long = unbound_long", "1234 = unbound_long");
 
         assertOptimizedEquals("10151082135029368 = 10151082135029369", "false");
+
+        assertOptimizedEquals("bound_varbinary = X'a b'", "true");
```

Comment:
These are invalid binary literals. According to the grammar rules in the spec, the string between the quotes needs to be made of pairs of hexadecimal characters optionally separated by one or more spaces.